crawlclima/redemet/rmet.py

```diff
@@ -154,6 +162,11 @@ def capture_date_range(station, date):
 
 
 def capture(station, date):
+    '''
```

Comment:
add docstring just for stating and date .. not for page and r_data

also use triple double quote (`"""your docstring"""`) instead of triple single quotes

for page and r_data you don't need to add it to the docstring